src/seat.c

```diff
@@ -143,6 +143,25 @@ configure_libinput(struct wlr_input_device *wlr_input_device)
 		wlr_log(WLR_INFO, "dwt configured");
 		libinput_device_config_dwt_set_enabled(libinput_dev, dc->dwt);
 	}
+
+	if (dc->send_events_mode < 0) {
+	} else if (dc->send_events_mode >= LIBINPUT_CONFIG_SEND_EVENTS_ENABLED) {
+		uint32_t possible_modes =
+			libinput_device_config_send_events_get_modes(libinput_dev);
+		enum libinput_config_status status;
+		if (dc->send_events_mode == LIBINPUT_CONFIG_SEND_EVENTS_ENABLED
+			|| possible_modes & dc->send_events_mode) {
```

Comment:
Our code style for this project requests that all continuation lines in a multi-line if statement should be indented two times to differentiate between the condition and the body. See the snippet I posted above for an example.